### PR TITLE
Ignore dynamic targets for monitoring test

### PR DESCRIPTION
**Pull request by sayanh**
**Description**

Changes proposed in this pull request:

- Following targets are only`up` when an application is created along with binding an event-service to a ns:
  - knative-eventing/knative-eventing-event-mesh-dashboard-broker
  - knative-eventing/knative-eventing-event-mesh-dashboard-httpsource
- When monitoring tests are executed in parallel to e2e tests, it does not make sense to check whether these targets are up or not cause there is no deterministic approach to that.
- These targets will eventually be tested here: https://github.com/kyma-project/kyma/issues/6457

**Related issue(s)**

See https://github.com/kyma-project/kyma/issues/8498

### CODE REVIEW

File: tests/integration/monitoring/test-monitoring.go [hunk 1]
```diff
@@ -152,13 +150,12 @@ func testTargetsAreHealthy() {
...
 			for _, target := range activeTargets {
-				// Ignoring the targets with certain labels
-				if hasAnyLabel(target.Labels, labelsToBeIgnored) {
+				if shouldIgnoreTarget(target.Labels) {
 					continue
 				}
 				if target.Health != "up" {
 					allTargetsAreHealthy = false
-					timeoutMessage += fmt.Sprintf("Target with job=%s and instance=%s is not healthy\n", target.Labels["job"], target.Labels["instance"])
+					timeoutMessage += fmt.Sprintf("Target with job=%s and instance=%s is not healthy, with labels=%v\n", target.Labels["job"], target.Labels["instance"], target.Labels)
```
Comment (marcobebway) [inline tests/integration/monitoring/test-monitoring.go:158]: A good candidate for a `strings.Builder`.

File: tests/integration/monitoring/test-monitoring.go [hunk 2]
```diff
@@ -170,9 +167,15 @@ func testTargetsAreHealthy() {
...
-		if target[l] == anyOf[l] {
+func shouldIgnoreTarget(target promAPI.Labels) bool {
+	var jobsToBeIgnored = []string{
+		// Note: These targets will be tested here: https://github.com/kyma-project/kyma/issues/6457
+		"knative-eventing/knative-eventing-event-mesh-dashboard-broker",
+		"knative-eventing/knative-eventing-event-mesh-dashboard-httpsource",
+	}
+
+	for _, j := range jobsToBeIgnored {
+		if target["job"] == j {
```
Comment (marcobebway) [inline tests/integration/monitoring/test-monitoring.go:178]: (nit) Does it make sense to use lower case `target["job"]`?

File: tests/integration/monitoring/test-monitoring.go (hunk 1 above)
Comment (sayanh) [reply tests/integration/monitoring/test-monitoring.go:158]: I agree. There are other occurrences of this pattern that need to be handled holistically in the file. Hence I would keep it as is for the sake of consistency.

File: tests/integration/monitoring/test-monitoring.go (hunk 2 above)
Comment (sayanh) [reply tests/integration/monitoring/test-monitoring.go:178]: I'd keep it as is for now. Let's take this offline.

**Pull request merged**